README.md

```diff
@@ -1,61 +1,80 @@
-# React Native example using HAAPI
+# React Native Example using HAAPI
 
 [![Quality](https://img.shields.io/badge/quality-demo-red)](https://curity.io/resources/code-examples/status/)
 [![Availability](https://img.shields.io/badge/availability-source-blue)](https://curity.io/resources/code-examples/status/)
 
-# Getting Started
+An example React Native app that uses the Curity Identity Server's Hypermedia API to perform an OIDC flow.\
+Authentication uses native screens without the need for an external browser.
 
-> **Note**: Make sure you have completed
-> the [React Native - Environment Setup](https://reactnative.dev/docs/environment-setup) instructions till "Creating a
-> new
-> application" step, before proceeding.
+## 1. Prepare a React Native Environment
 
+First make sure you have completed the [React Native - Environment Setup](https://reactnative.dev/docs/environment-setup).\
+Follow the React Native instructions and ensure that you can run a basic app.
 
-To install necessary dependencies, run
+## 2. Deploy the Curity Identity Server
 
-```bash
-npm install
-```
+Start with a local automated deployment to ensure that you understand the technical setup.\
+You can then apply the same configuration to deployed environments.\
+First ensure that the local computer has these prerequisites:
+
+- A Docker engine.
+- The `envsubst` tool, e.g with `brew install gettext`.
+- The `jq` tool, e.g with `brew install jq`.
+
+Copy a `license.json` file for the Curity Identity Server into the root folder.\
+Then deploy the system, and indicate how connected emulators or devices call the Curity Identity Server.\
+The following example uses the ngrok tool to provide an internet HTTPS URL:
 
-For iOS, you also need to run
 ```bash
-cd ios
-pod install
+export USE_NGROK='true'
```

Comment:
We could list ngrok as an optional prerequisite and maybe link somewhere to installation instructions.